.gitlab-ci.yml

```diff
@@ -326,9 +326,20 @@ benchmarks-statemint:
     - ./scripts/benchmarks-ci.sh assets westmint ./artifacts
     - export BRANCHNAME="weights-statemint-${CI_COMMIT_BRANCH}"
     - *git-commit-push
+    # create PR to ${CI_COMMIT_BRANCH}
+    - curl -u ${GITHUB_USER}:${GITHUB_TOKEN}
+      -d '{"title":"[benchmarks] Update weights for statemine/t","body":"This PR is generated automatically by CI.","head":"'${BRANCHNAME}'","base":"'${CI_COMMIT_BRANCH}'"}'
```

Comment:
Are other runtimes done in another location ?
We need weights for `westmint`, `rococo-parachain`, etc...

Reply:
Yeah, the name of the job is confusing after `benchmarks-collectives` was added. `benchmarks-statemint` is actually run for `statemine`, `statemint` and `westmint`. Do you have a suggestion for renaming?

Reply:
Sure, in the new structure, all of those are under the `assets` category.
See https://github.com/paritytech/cumulus/tree/master/parachains/runtimes